editions/tw5.com/tiddlers/filters/syntax/Map Filter Run Prefix (Examples).tid

```diff
@@ -20,12 +20,27 @@ Get the tags of all tiddlers tagged `Widget:`
 <<.operator-example 3 "[tag[Widgets]] :map:flat[tagging[]] :and[!is[blank]unique[]]">>
 <<.tip "Without the `flat` suffix the `:map` filter run only returns the first result for each input title">>
 
+!! Comparison between `:map` with and without the `flat` suffix
+
+The `:map` filter run will return at least as many outputs as given in the input.  By default one input item will result in exactly one output item. When the filter run transforms an input item into an empty result, the output for that item will be an empty string. When the filter run transforms an input item into multiple items, only the first item will appear in the output. This behavior can be overridden by providing the `flat` suffix. The `flat` suffix will cause all the items to appear in the output.
+
+Any `:map` filter run can be expressed with `:map:flat` by adding a call to the [[first operator|first Operator]] at the end of the filter run.
```

Comment:
@btheado the rest looks good but this line is extremely confusing and I think should be dropped.

Reply:
Removed